api/registrations/views.py

```diff
@@ -860,6 +861,54 @@ def get_object(self):
         return obj
 
 
+class LinkedRegistrationsList(JSONAPIBaseView, generics.ListAPIView, RegistrationMixin):
```

Comment:
Done. Both instances of LinkedRegistrationsList and both instances of LinkedNodesList now subclass BasLinkedList.